### PR TITLE
Fix: Install missing is-ci package

### PR DESCRIPTION
**Pull request by MansurAliKoroglu**
## Description

We use `is-ci` in `package.json` scripts but package was missing.

## Checklist

- [x] My changes are documented
- [x] My changes are tested
- [x] Quality tools pass locally with my changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "eslint-plugin-security": "1.4.0",
     "eslint-plugin-sonarjs": "0.8.0-125",
     "husky": "7.0.1",
+    "is-ci": "2.0.0",
```
Comment (UgurCanTiryaki) [inline package.json:33]: The latest version of is-ci is `3.0.0`. Why do we use version `2.0.0`?

Comment (MansurAliKoroglu) [reply package.json:33]: I did not manually install `2.0.0` Just did `npm install --save-dev` and apparently it installed `2.0.0`. Probably when I create this PR, version `3.0.0` was not released to `npm`. Updating it now. Good catch.

Comment (MansurAliKoroglu) [reply package.json:33]: That's not the reason. `npm install --save-dev` still installs `2.0.0` Don't know why.

Comment (UgurCanTiryaki) [reply package.json:33]: It's not the same for me. `npm install --save-dev` installs version `3.0.0` in my case.

Comment (MansurAliKoroglu) [reply package.json:33]: Hmm my npm version is `7.20.0` are we using the same version?

Comment (UgurCanTiryaki) [reply package.json:33]: No, mine is `6.14.13`. Yours is newer than mine. Then maybe this is not a bug but a new feature.

Comment (MansurAliKoroglu) [reply package.json]: I upgraded npm to `7.20.3` and gave it a try. It installed `3.0.0` this time.

**Review by UgurCanTiryaki — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (MansurAliKoroglu) [reply package.json]: It's probably fixed with this one. https://github.com/npm/cli/pull/3588

**Pull request merged**